parquet-column/src/main/java/org/apache/parquet/column/values/rle/RunLengthBitPackingHybridDecoder.java

```diff
@@ -56,6 +58,17 @@ public RunLengthBitPackingHybridDecoder(int bitWidth, ByteArrayInputStream in) {
     Preconditions.checkArgument(bitWidth >= 0 && bitWidth <= 32, "bitWidth must be >= 0 and <= 32");
```

Comment:
Confirming this was fixed, github isn't hiding the comment, but there is only one constructor that works for both of the previous cases.